src/Tasks/Microsoft.NET.Build.Tasks/ResolvedFile.cs

```diff
@@ -2,15 +2,23 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 
 using System.IO;
-
+using System;
 namespace Microsoft.NET.Build.Tasks
 {
+    public enum AssetType
```

Comment:
NuGet doesn't seem to have an enum for AssetType, do we need one?  We are just turning strings into this enum, and then back into strings to put on Item metadata.  Is the enum necessary?